client/public/index.html

```diff
@@ -25,7 +25,10 @@
       Link to CDN for microapps.js library?
     -->
     <script src="https://microapps.google.com/apis/v1alpha/microapps.js"></script>
-    <script id="mapPlacesAPIScript" src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBkGOvn8gUfLrECXPQ5KFNCHUqoXr3mzCc&libraries=places"></script>
+    <script
+      id="mapPlacesAPIScript"
+      src="https://maps.googleapis.com/maps/api/js?key=%REACT_APP_GOOGLE_MAPS_API_KEY%&libraries=places"
```

Comment:
This key is only embedded during build time, consequently, this currently breaks `yarn dev`

Reply:
Might be able to workaround via loading from `Actions.ts` by using js to form the script src string after grabbing it from `process.env` instead.

Reply:
This works for me - not facing any breakages. I inspected the code and see the correct values being passed through - can you check again?

Reply:
yes - everytime you change the key/config though, you need to re-run yarn dev.

Reply:
my bad, checked this again, works for me too :/ maybe I forgot to export the env variables prior to running `yarn create-env` the last time